server/data/apps.json

```
@@ -1989,5 +2005,22 @@
         "elementURL": "https://bitbadger.solutions/open-source/myweblog/rss.html#podcast-feeds"
       }
     ]
-  } 
+  },
+  {
+    "appName": "podCloud",
+    "appType": [
+      "hosting"
```

```suggestion
      "hosting",
      "app",
      "podcast player"
```

We're also a podcast player web app :-)

---

Thanks @benjaminbellamy for adding us ! 🙂

---

It's for webapps also. CastCoverage, curiocaster, cryptostream.online, etc are set as app with Web only as platform in the same file